content/systems/filecoin_files/data_transfer/_index.md

```diff
@@ -50,61 +58,41 @@ or potentially the same round trip, where the requesting party implicitly agrees
 {{<svg src="push-flow.mmd.svg" title="Data Transfer - Push Flow">}}
 
 1. A requestor initiates a Push transfer when it wants to send data to another party.
-2. The requestors' data transfer module will send a push request to the responder along with the data transfer voucher. It also puts the data transfer in the scheduler queue, meaning it expects the responder to initiate a transfer once the request is verified
-3. The responder's data transfer module validates the data transfer request via the Validator provided as a dependency by the responder
-4. The responder's data transfer module schedules the transfer
-5. The responder makes a GraphSync request for the data
-6. The requestor receives the graphsync request, verifies it's in the scheduler and begins sending data
-7. The responder receives data and can produce an indication of progress
-8. The responder completes receiving data, and notifies any listeners
-
-The push flow is ideal for storage deals, where the client initiates the push
-once it verifies the deal is signed and on chain
+2. The requestors' data transfer module will send a push request to the responder along with the data transfer voucher.
+3. The responder's data transfer module validates the data transfer request via the Validator provided as a dependency by the responder.
+4. The responder's data transfer module initiates the transfer by making a GraphSync request.
+5. The requestor receives the GraphSync request, verifies that it recognises the data transfer and begins sending data.
+6. The responder receives data and can produce an indication of progress.
+7. The responder completes receiving data, and notifies any listeners.
 
-### Pull Flow
+The push flow is ideal for storage deals, where the client initiates the data transfer straightaway
+once it verifies the deal is signed and on chain.
```

Comment:
Eek not sure how this misunderstanding happened, but data transfer happens BEFORE the deal is on chain.

```suggestion
once the provider indicates their intent to accept and publish the client's deal proposal.
```